frontend/src/components/SupportedAppTitle.tsx

```diff
@@ -34,7 +41,13 @@ const SupportedAppTitle: React.FC<SupportedAppTitleProps> = ({ odhApp, showProvi
 
   return (
     <CardTitle>
-      <span style={{ verticalAlign: 'text-bottom' }}>{title}</span>
+      {onClick ? (
```

Comment:
Looks like this doesn't work here. Whether the title is a button style or not doesn't only depend on the `onClick` function, but also depends on the props we pass to the `Card` component. See https://www.patternfly.org/components/card#clickable-and-selectable as an example. That's why we don't pass `onClick` to the card on the `Enabled` page but we still get a link button style on the title (we want the title to be just bold plain text here)

<img width="377" alt="Screenshot 2023-11-01 at 2 14 12 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/7146c999-491c-49be-829d-2ad28f2dfdc3">

Reply:
Thank you @DaoDaoNoCode , good point. Added the [`isPlain` prop](https://github.com/opendatahub-io/odh-dashboard/pull/2179/files#diff-3ee554a22ca13ed93335b8b80c60f6e9b8915801192fb0e847a36833108617acR201) to prevent rendering a link there